consul/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+services:
+  consul:
+    image: "consul:0.8.0"
+    ports:
+      - "8500:8500"
+  service:
```

Comment:
Looks like the extra newline between services got lost in the shuffle. Could you add a line break between lines 6 & 7 and also 24 & 25? Also please rename this file to be `consul/docker-compose.yml`